packages/react-devtools-core/README.md

```diff
@@ -1,25 +1,26 @@
 # `react-devtools-core`
 
-A standalone React DevTools implementation.  
+A standalone React DevTools implementation.
 
-This is a low-level package.  
+This is a low-level package.
 **If you're looking for the Electron app you can run, use `react-devtools` package instead.**
 
 ## Exports
 
 ## `require('react-devtools-core').connectToDevTools(options)`
 
-This is similar to `require('react-devtools')` in another package but providing more control.  
```

Comment:
Spaces are significant in Markdown. Here, they cause the line to break. Could you please remove this change?